py_mmd_tools/xml_util.py

```diff
@@ -0,0 +1,58 @@
+"""
+Utility tool to work on mmd xml files.
+ License:
+     This file is part of the S-ENDA-Prototype repository (https://github.com/metno/py-mmd-tools).
```

Comment:
Please change to License: `This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)`